gateway-home/modules/ROOT/pages/index.adoc

```diff
@@ -1,53 +1,85 @@
+Using Mule Gateway, any user can, for example, apply a basic authentication policy on top of a Mule application, enrich an incoming/outgoing message, or add any other complex capability to an API without having to write any code.
 
 If you want to apply policies and generate analytics info, you must use either:
 
 * An API defined in a Mule application.
 * A Mule application that proxies your existing backend implementation.
 
-For more information, see xref:mule-gateway::mule-gateway-capabilities-mule4.adoc[Mule Gateway Capabilities].
+For more information, see xref:mule-gateway::mule-gateway-capabilities-mule4.adoc[].
+
+== Flex Gateway vs. Mule Gateway
+
+While Flex Gateway is a high-performance Envoy-based gateway with a small footprint, MuleSoft also provides a simple Mule Gateway for Mule applications.
+
+Mule runtime engine, based on Java Spring, includes an embedded Mule Gateway. By using this gateway, you can, for example:
+
+* Apply a basic authentication policy on top of a Mule application.
+* Enrich an incoming or outgoing message.
+* Add any other complex capability to an API without having to write any code.
+
+Mule Gateway protects a single Mule API. The key advantage is that it's easy for Mule app developers to provide basic endpoint protection.
+
+You can also configure Mule in Anypoint Runtime Manager as a CloudHub Proxy application, protecting multiple backends. However, in this case the gateway cannot be easily scaled to support high-availability and high-performance applications. Further, Mule consumes more computes or vCores than the smaller-footprint Flex Gateway.
+
+Building custom policies on Mule Gateway is similar to building an application with Java using the Mule DSL. Building a custom policy in Flex Gateway is based on Envoy-provided Rust WASM SDKs. A Mule Gateway policy cannot be reused in Flex Gateway and vice versa, because the underlying architectures are fundamentally different.
+
+MuleSoft recommends you choose Flex Gateway for high-availability and high-performance applications.
+
+[%header%autowidth.spread,cols="a,a,a"]
+|===
+| | Flex Gateway | Mule Gateway
+
+| *Summary*
+| Envoy-based API gateway to secure all APIs, Mule and non-Mule, running anywhere.
+| Java-based API gateway embedded into Mule.
 
-== Anypoint Service Mesh
+| *Tech Stack*
+| Underlying engine build upon Envoy, and FluentBit for logging.
+| Java Spring Application embedded into the Mule.
 
-Anypoint Service Mesh enables you to extend your microservices network by including your non-MuleSoft applications into the Anypoint Platform sphere. You can then manage and secure your application network, including your non-Mule applications, seamlessly from a single pane.
+| *Use Case*
+| High-performant and high-availability. Secure any API anywhere.
+| Secure a Mule API "Basic Endpoint", or enable a dedicated proxy as an embedded library in a Mule. Also available as a Mule proxy application in CloudHub.
```

Comment:
Is Basic Endpoint a proper noun?

"in a Mule instance."

Reply:
For the multiple sentences/fragments in this table, can you use a bullet list? It would be easier to scan (and more straightforward to punctuate... or not).

Reply:
Fixed